platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/eventhandling/TransactionPool.java

```diff
@@ -39,58 +36,63 @@
  * Store a list of transactions created by self, both system and non-system, for wrapping in the next event to be
  * created.
  */
-public class EventTransactionPool implements TransactionPool, TransactionSupplier {
+public class TransactionPool implements TransactionSupplier, EventCreationRule {
```

Comment:
nit: Line 218 of `SwirldsPlatform` still mentions "event transaction pool"

Reply:
fixed